birdhouse/deployment/deploy.sh

```diff
@@ -67,7 +67,7 @@ else
     shift
 fi
 
-BIRDHOUSE_LOCAL_ENV="${1:-${BIRDHOUSE_LOCAL_ENV:-"${COMPOSE_DIR}/env.local"}}"
+BIRDHOUSE_LOCAL_ENV="${2:-${BIRDHOUSE_LOCAL_ENV:-"${COMPOSE_DIR}/env.local"}}"
```

Comment:
I'll test but probably not it because there is a `shift` right above, meaning `$2` becomes `$1`.

Reply:
I tested, it's not the fix.